gitops/ckcp/base/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kcp
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Mi
```

Comment:
What is the use case?
- production/staging environment
- CI
- development

I would guess the 3 of them are valid use cases. 10Mi may be fine for CI and dev but surely not for production and staging. This also applies to the CPU / memory requests and limits. We need a README, which describes:

- when to use this configuration
- how to use it with each use case
- what needs to be configured, tuned
- sane defaults for each use case

Reply:
Hey @fgiloux , to my understanding, this is meant solely for the development purposes. And if it's going to be used for production/staging/CI, we never had this discussed. 
So, following on that -
1. This configuration in its current state of the manifests is meant for development only.
2. Using the `ckcp/gitops.sh` --> Presently, this script is being worked on and tracked in another Jira issue.
3. Host address needs to be configured properly according to ingress/route suffix. This dependency is supposed to go away with `ckcp/gitops.sh`.
4. We're good with defaults for now. In future, we may need to tweak `./gitops/ckcp/base/deployment.yaml` a/c to KCP image. 

If this seems okay, I can put this properly in a README.

Reply:
@RAMESSESII2
1. thanks for the clarification. To be honest I don't see how it improves my dev experience.
2. ckcp/gitops.sh we are moving away from this script
3. I know

When you have validated, please update DEVELOPMENT.md to describe the option.

Looking at my first 2 comments, they are marked as resolved but the code has not changed. Is it pending on a push from you?

Reply:
Yes, I'm gonna make the required changes and push them all together.